docs/features/techdocs/how-to-guides.md

```diff
@@ -759,11 +759,11 @@ const techdocsCustomerBuildStrategy = createBackendModule({
 // Other plugins...
 
 /* highlight-add-start */
-backend.add(import('@backstage/plugin-search-backend-module-catalog/alpha'));
-backend.add();
+backend.add(import('@backstage/plugin-techdocs-backend/alpha'));
+backend.add(techdocsCustomerBuildStrategy());
```

Comment:
```suggestion
backend.add(techdocsCustomBuildStrategy());
```

Should we change this too? The initial `const` above would need to be updated too